trunk/SUAVE/Attributes/Gases/Air.py

```diff
@@ -100,16 +101,10 @@ def compute_speed_of_sound(self,T=300.,p=101325.,variable_gamma=False):
         if variable_gamma:
             g = self.compute_gamma(T,p)
         else:
-            g = 1.40
+            g = 1.4*np.ones_like(T)

```

Comment:
Any issues if you leave this and T is a scalar?

Reply:
Not that I have found